README.md

```diff
@@ -166,6 +166,40 @@ There are two ways to override the behaviour:
 
   You can also set the operator to ignore the services by default and only manage them when the annotation is true. `dev.inlets.manage=true` with the flag `-annotated-only`
 
+## Running in combination with other LoadBalancer controllers
+
+By default the operator will create a tunnel for every LoadBalancer service, which can cause issues when running in combination with other LoadBalancer controllers like MetalLB.
+
+When running the inlets-operator with the `-annotated-only` flag, the inlets-operator will ignore all LoadBalancer services, unless explicitly annotated with `dev.inlets.manage=true`, and will not compete against the other LoadBalancer controllers.
+
+The operator is still able to expose services in this mode, by using a ClusterIP service with a Tunnel resource.
```

Comment:
In this example, how do you find out the server IP?

Reply:
funny enough, the ClusterIP service is actually updated with the server IP by the inlets-operator

Reply:
Can you explain that and show an example command from kubectl?

Reply:
This section may be better under `Annotations and ignoring services`

Or maybe as the primary title for `Annotations and ignoring services` and then we have three sub-titles?